clickqt/widgets/optiongrouptitlewidget.py

```diff
@@ -41,3 +55,53 @@ def set_value(self, value: t.Any):
 
     def get_widget_value_cmdline(self) -> str:
         return ""
+
+    def option_group_visibility(self):
+        """Function to determine if all the necessary widgets are already collapsed or not."""
+        return all(
+            self.control_instance.widget_registry[self.key]
+            .get(grouped_option)
+            .is_collapsed
+            is False
+            for grouped_option in self.groups
+        )
```

Comment:
id use a negated any instead of all ... is False
i would also call the function accordingly, so that you can tell what the single line this function is actually does based on its name